### PR TITLE
feat: [STO-6544]: add anchore enterprise step

### PR DESCRIPTION
**Pull request by atmollo-harness**
### Summary

Add new anchore step to step pallet

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by atmollo-harness** [comment]
retrigger titlecheck

**Comment by atmollo-harness** [comment]
retrigger cypress-cv

**Comment by atmollo-harness** [comment]
retrigger cypress-pipeline

File: src/modules/85-sto/components/PipelineSteps/AnchoreStep/AnchoreStep.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  initialValues: AnchoreStepData
+  template?: AnchoreStepData
+  path?: string
+  isNewStep?: boolean
+  readonly?: boolean
+  stepViewType: StepViewType
+  onUpdate?: (data: AnchoreStepData) => void
+  onChange?: (data: AnchoreStepData) => void
+  allowableTypes: AllowedTypes
+  formik?: any
```
Comment (jsentongo) [inline src/modules/85-sto/components/PipelineSteps/AnchoreStep/AnchoreStep.tsx:35]: formik?: FormikProps<AnchoreStepData> maybe?

**Review by jsentongo — COMMENTED** [review]
LGTM

**Comment by atmollo-harness** [comment]
retrigger jest

File: src/modules/85-sto/components/PipelineSteps/AnchoreStep/AnchoreStep.tsx (hunk 1 above)
Comment (atmollo-harness) [reply src/modules/85-sto/components/PipelineSteps/AnchoreStep/AnchoreStep.tsx]: fixed

**Review by jsentongo — DISMISSED** [review]

**Review by jsentongo — APPROVED** [review]

**Review by harness-dc — APPROVED** [review]

**Pull request merged**